osCam/userconfig/models.py

```diff
@@ -3,24 +3,55 @@
 from django.contrib.auth.models import User
 
 
-# Create your models here.
 class Network(models.Model):
     home_ip_address = models.CharField(max_length=32, null=True, blank=True)
     camera_ip_address = models.CharField(max_length=32, null=True, blank=True)
     user = models.ForeignKey(User, on_delete=models.CASCADE, blank=True, null=True)
     # network = models.ForeignKey(RaspberryPi, on_delete=models.DO_NOTHING, null=True, blank=True)
 
+
 class RaspberryPi(models.Model):
     # User can have Many RaspberryPie (1:Many)
     user = models.ForeignKey( User, on_delete=models.CASCADE, null=True, blank=True )
     #network can have many RaspberryPie (1:Many) (we could do manyToMany but assuming we're coding MVP model)
     network = models.ForeignKey(Network, on_delete=models.DO_NOTHING, null=True, blank=True)
     model_num = models.IntegerField(blank=True, null=True)
     model_name = models.CharField(max_length=40,blank=True, null=True)
+    username = models.CharField(max_length=40, default='admin')
+    password = models.CharField(max_length=40, default='admin')
+
 
 class Camera(models.Model):
     # Rasberry Pi can have Many Camera (1:Many)
     user = models.ForeignKey( User, on_delete=models.CASCADE, null=True, blank=True )
     raspberry_pi = models.ForeignKey( RaspberryPi , on_delete=models.CASCADE, null=True, blank=True)
     model_num = models.IntegerField(blank=True, null=True)
     model_name = models.CharField(max_length=40,blank=True, null=True)
+    camera_index = models.IntegerField(default=0, unique=True, editable=False)
+    device_name = models.CharField(max_length=40, default='Camera')
```

Comment:
We could try to auto populate a lot of the camera/hardware/networking info

Reply:
Absolutely. A lot of things that will be programmatically discovered is just getting dummy values for now. But anything we recognize that could be automated would make great improvements for the remainder of the semester.